cmd/dfget/app/root.go

```diff
@@ -72,38 +71,38 @@ func init() {
 func runDfget() error {
 	// initialize logger
 	if err := initClientLog(); err != nil {
-		util.Printer.Println(fmt.Sprintf("init log error: %v", err))
+		fmt.Printf("init log error: %v\n", err)
 		return err
 	}
 
 	if err := transParams(); err != nil {
-		util.Printer.Println(err.Error())
+		fmt.Println(err.Error())
```

Comment:
I think that it's better to encapsulate the stdout, it's convenient for us to add some extra information in the future.

Reply:
Finished that. Thanks for your explanation. @lowzj